### DISCUSSION TITLE
How do I make use of local Storage to store the hours, minutes and seconds?

### DISCUSSION THREAD

**Discussion by jomefavourite** [ndresx/react-countdown#148] [Q&A] [answered]
I've been able to make use of the awesome library which saved me some stress, but I'll love it if I could get an example where the props `hours`, `minutes`, `seconds` will be stored on the local storage.

**Comment by ndresx** [comment, marked as answer]
Hi, although you might not need an answer to this anymore given my late reply (sorry for this), one way to store them in local storage (for whatever reason) could be via https://github.com/ndresx/react-countdown#ontick callback and the browser's native local storage API.

The countdown itself though has nothing to do with the local storage.

**Reply by jomefavourite** [reply]
Okay, thanks for the feedback
